docs/nginx.md

```diff
@@ -0,0 +1,42 @@
+# Nginx Receiver
+
+The `nginx` receiver can fetch stats from a NGINX instance using the mod_status endpoint.
+
+
+## Prerequisites
+
+[ngx_http_stub_status_module](http://nginx.org/en/docs/http/ngx_http_stub_status_module.html) must be enabled on the NGINX instance. This may require editing the NGINX configuration.
+
+
+## Configuration
+
+| Field                 | Default                   | Description |
+| ---                   | ---                       | ---         |
+| `type`                | required                  | Must be `nginx`. |
+| `endpoint`            | `http://localhost/status` | The url exposed by the NGINX stats module. |
+| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    mynginxmetrics:
+      type: nginx
+      endpoint: http://localhost:80/status
+      collection_interval: 30s
+  service:
+    pipelines:
+      nginxpipeline:
+        receivers:
+          - nginxmetrics
+```
+
+## Metrics
+
+| Metric                     | Data Type | Unit        | Labels | Description |
+| ---                        | ---       | ---         | ---    | ---         | 
+| nginx.requests             | sum       | requests    |        | Total number of requests made to the server. |
```

Comment:
At Ops Agent level, the metrics documentation should be about the Google Monitoring format instead of OTEL, so that users can easily locate it in Metrics Explorer in the UI. Should this be something like `XXX.googleapis.com/nginx...`? @jsuereth might know more details of the expectation.

Reply:
I've prepended each with `custom.googleapis.com`. I think this makes sense, but depends on resolution to [this comment](https://github.com/GoogleCloudPlatform/ops-agent/pull/153#discussion_r694355052).

Reply:
These are now documented correctly as `workload.googleapis.com/nginx.foo`.